src/queue.js

```diff
@@ -88,6 +88,9 @@ Queue.prototype.addItem = function(item, callback, originalError, originalItem)
   }
   this._maybeLog(item, originalError);
   this.removePendingItem(originalItem);
+  if (!this.options.transmit) {
```

Comment:
- This might be the right place, or perhaps this should go in `_makeApiRequest` - what do you think?
- Looks like we are no longer calling the callback - is that intentional?

Reply:
This location is (I believe) after everything the caller would want to execute without sending the request. At entry of `_makeApiRequest` is logically equivalent. Any later would engage rate limiting. (And maybe that's desired? I'm not sure, but I thought not.)

I skipped the callback, since this case neither has a response nor produces an error. But it probably needs to be called in order for async/Promise wrappers to work correctly. I imagine other situations as well. I'll add it.